BookingsApi/BookingsApi.Infrastructure.Services/AsynchronousProcesses/FirstdayOfMultidayHearingAsynchronousProcess.cs

```diff
@@ -13,22 +12,13 @@ public interface IFirstdayOfMultidayBookingAsynchronousProcess
     public class FirstdayOfMultidayHearingAsynchronousProcess : IFirstdayOfMultidayBookingAsynchronousProcess
     {
         private readonly IEventPublisherFactory _publisherFactory;
-        private readonly IFeatureToggles _featureToggles;
-        public FirstdayOfMultidayHearingAsynchronousProcess(IEventPublisherFactory publisherFactory, IFeatureToggles featureToggles)
+        public FirstdayOfMultidayHearingAsynchronousProcess(IEventPublisherFactory publisherFactory)
         {
             _publisherFactory = publisherFactory;
-            _featureToggles = featureToggles;
         }
 
         public async Task Start(VideoHearing videoHearing)
         {
-            if (!_featureToggles.UsePostMay2023Template())
-            {
-                await _publisherFactory.Get(EventType.CreateAndNotifyUserEvent).PublishAsync(videoHearing);
```

Comment:
If the old events are part of the old flow then please remove them so they're not loitering in the code base unused

Reply:
I am not sure what you mean. I have removed all the code for the feature flag